pulp_smash/cli.py

```diff
@@ -276,6 +282,9 @@ def run(self, args, sudo=False, **kwargs):
         kwargs.setdefault("retcode")
         logger.debug("Running %s cmd (sudo:%s) - %s", args, sudo, kwargs)
 
+        if self._podname:
+            args = ("kubectl", "exec", self._podname) + tuple(args)
```

Comment:
We need -- (dash dash) at the end also.

So that we can run commands with arguments.